common/changes/@itwin/core-frontend/nam-dta-electron-fix_2023-07-26-19-35.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Use import instead of require",
```

```suggestion
      "comment": "",
```
since this will be user facing, dont really need to let them know, they wont care